Microsoft.Toolkit.Uwp.SampleApp/App.xaml

```diff
@@ -16,7 +16,7 @@
 				<ResourceDictionary Source="ms-appx:///Styles/Generic.xaml" />
 
 				<!-- WinUI -->
-				<not_wasm:XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
```

Comment:
@davidjohnoliver or @jeromelaban  any idea why there's a not_wasm for the winui resources?

Reply:
It's a recent PR https://github.com/unoplatform/Uno.WindowsCommunityToolkit/pull/105 - I gather it was package-size-related but @jeromelaban may have more details?